src/main.rs

```diff
@@ -0,0 +1,113 @@
+use crate::gemfile_lock::{BundlerVersion, GemfileLock, GemfileLockError, RubyVersion};
```

Comment:
```suggestion
#![warn(unused_crate_dependencies)]
#![warn(clippy::pedantic)]
#![allow(clippy::module_name_repetitions)]

use crate::gemfile_lock::{BundlerVersion, GemfileLock, GemfileLockError, RubyVersion};
```

Reply:
Note: You'll also need to work around the false positives from the integration tests, like so:
https://github.com/heroku/procfile-cnb/blob/76e5aa0d5467c46da076ab4ddde6c0b392c3ad20/src/main.rs#L23-L24